vizro-core/examples/visual-vocabulary/custom_charts.py

```diff
@@ -65,80 +63,87 @@ def sankey(data_frame: pd.DataFrame, source: str, target: str, value: str, label
         labels (list[str]): A list of labels for the nodes.
 
     Returns:
-        go.Figure: A Plotly Figure object representing the Sankey chart.
+        go.Figure: A Plotly Figure object of the Sankey chart.
 
     For detailed information on additional parameters and customization, refer to the Plotly documentation:
     https://plotly.com/python/reference/sankey/
 
     """
-    fig = go.Figure(
-        data=[
-            go.Sankey(
-                node={
-                    "pad": 16,
-                    "thickness": 16,
-                    "label": labels,
-                },
-                link={
-                    "source": data_frame[source],
-                    "target": data_frame[target],
-                    "value": data_frame[value],
-                    "label": labels,
-                    "color": "rgba(205, 209, 228, 0.4)",
-                },
-            )
-        ]
+    return go.Figure(
+        data=go.Sankey(
+            node={
+                "pad": 16,
+                "thickness": 16,
+                "label": labels,
+            },
+            link={
+                "source": data_frame[source],
+                "target": data_frame[target],
+                "value": data_frame[value],
+                "label": labels,
+                "color": "rgba(205, 209, 228, 0.4)",
+            },
+        ),
+        layout={"barmode": "relative"},
     )
-    fig.update_layout(barmode="relative")
-    return fig
 
 
 @capture("graph")
-def column_and_line(data_frame: pd.DataFrame, x: str, y_column: str, y_line: str) -> go.Figure:
+def column_and_line(
+    data_frame: pd.DataFrame,
+    x: Union[str, pd.Series, list[str], list[pd.Series]],
+    y_column: Union[str, pd.Series, list[str], list[pd.Series]],
+    y_line: Union[str, pd.Series, list[str], list[pd.Series]],
+) -> go.Figure:
```

Comment:
This is where this approach hits a limit, and we cannot pass kwargs twice - an issue we also encountered in the charts of old, just mentioning this again, as we would start to have inconsistencies between the charts etc

Reply:
Yes, I spent a long timed thinking about this plot and found it very awkward for exactly that reason! As I saw it the options were:
* declare one chart to be the "primary" one and pass `**kwargs` through to that. But here the column and line charts are really on equal standing so it would be an arbitrary choice which was the primary chart
* pass the same `**kwargs` through to both the column and line charts. This limits `kwargs` to only work with arguments that are common between both and always passes the same value to both. I'm not sure in practice if that is fine or makes it useless
* have arguments `column_kwargs` and `line_kwargs`. I _think_ this is actually ok (it's not so different from how other things are handled in plotly) but didn't think it was worth doing here since I'm not 100% sure